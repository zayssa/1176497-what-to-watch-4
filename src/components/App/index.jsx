import React from "react";
import Main from "../Main/index.jsx";


const App = (data) => {
  return (
    <Main {...data} />
  );
};

export default App;
