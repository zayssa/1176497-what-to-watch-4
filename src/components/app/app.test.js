import React from "react";
import renderer from "react-test-renderer";
import App from "./app.jsx";

it(`Render App`, () => {
  const data = {
    name: `atata`,
    genre: `bugaga`,
    date: `2026`,
    films: [
      {
        id: 1,
        title: `Titanic`,
        poster: `img/titanic.jpg`
      },

      {
        id: 2,
        title: `Back to the future`,
        poster: `img/back.jpg`
      },

      {
        id: 3,
        title: `Academy`,
        poster: `img/academy.jpg`
      },

      {
        id: 4,
        title: `Moana`,
        poster: `img/moana.jpg`
      },
    ]
  };

  const tree = renderer
    .create(<App
      {...data}
    />)
    .toJSON();

  expect(tree).toMatchSnapshot();
});
