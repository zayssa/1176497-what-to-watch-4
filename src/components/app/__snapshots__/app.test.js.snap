// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render App 1`] = `
<div
  style={
    Object {
      "alignItems": "center",
      "display": "flex",
      "height": "100vh",
      "justifyContent": "center",
    }
  }
>
  Loading...
</div>
`;
